testing/srt-test-live.cpp

```diff
@@ -350,6 +350,19 @@ int main( int argc, char** argv )
     bool internal_log = Option("no", "loginternal") != "no";
     bool skip_flushing = Option("no", "S", "skipflush") != "no";
 
+    // Print format
+    string pf = Option("default", "pf", "printformat");
+    if (pf == "json")
+    {
+        transmit_printformat_json = true;
+    }
+    else if (pf != "default")
```

Comment:
Support for CSV?

Reply:
There was some other PR that has provided advanced formats for it, but there was also much more. I think it was at the redundancy. This is only for development and I'm possibly the only user of this tool so far, maybe better leave it for future.